workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml

```diff
@@ -303,8 +303,8 @@ mapaction_rivers_length:
   endpoint: elements/length
   filter: waterway in (riverbank,river) and type:way
 
-ideal_vgi_infrastructure:
-  name: IDEAL-VGI Infrastructure
+basic_infrastructure:
+  name: Basic Infrastructure
   description: Objects related to infrastructure
   endpoint: elements/length
   filter: (aerialway=* or aeroway=* or highway=* or power=* or railway=* or telecom=*) and geometry:line
```

Comment:
@SlowMo24 what would be a good name for this layer? Basic Infrastructure is a bit broad of a term is it not? Because its only lines, so infrastructures which are polygon or points like power boxes are not included in this layer.
Maybe a fitting name would be `Infrastructure Lines`? Also is there a source we can point to?

Reply:
Yes, `Infrastructure Lines` sounds good. There is no source to point to except https://gitlab.gistools.geog.uni-heidelberg.de/giscience/ideal-vgi/osm-element-vectorisation/-/blob/main/src/osm_element_vectorisation/resources/misc/tagGrouping.json